tests/console/yast2_clone_system.pm

```diff
@@ -38,7 +38,25 @@ sub run {
 
     # Check and upload profile for chained tests
     upload_asset "/root/autoinst.xml";
-    if (script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml') {
+
+    unless (is_opensuse) {
+        my $sle15_repo;
+        if (is_sle('>15')) {
+            my $sp = (split(/-/, get_var('VERSION')))[-1];
+            $sle15_repo = "REPO_SLE15_" . "$sp" . "_MODULE_DEVELOPMENT_TOOLS";
+        }
+        else {
+            $sle15_repo = 'REPO_SLE15_MODULE_DEVELOPMENT_TOOLS';
+        }
+
+        my $devel_repo = get_var(is_sle('>=15') ? $sle15_repo : 'REPO_SLE_SDK');
```

Comment:
I guess we should use get_required_var here, so it's more easy to see what is the problem if it fails.